cloudflare/workers-rs
# Streaming support

Adds basic streaming support Requests and Responses. This PR is currently blocked on #142 because the `wasm-streams` library used requires a snippet, so CI will fail until that PR has landed and released.


# Manual Testing

Currently to test this PR manually you need to:

1. Checkout to the Js Snippets PR
2. Install `worker-build` system-wide
3. Checkout to this PR
4. Comment out the build command from `worker-sandbox/wrangler.toml`
5. Manually compile the worker with `worker-build --release`
6. Start the worker with `miniflare -c ./wrangler.toml --no-cf-fetch --no-update-check`
7. In another terminal session run `cargo test xor`
